INSTALL.asciidoc

```diff
@@ -25,16 +36,20 @@ sudo zypper in libtheora-devel opencv-devel fftw3-devel libsndfile-devel pkg-con
 autoreconf -f -i
 ./configure
 make
+make install
 ----------------
++
+If you do not run `make install` then you need to ensure Perl can find
+`ppmclibs/blib/arch/auto/tinycv.so` and `ppmclibs/tinycv.pm` some other way.
 5.  get the test cases and the needles.
 +
 The needles have to be a subdir of the test cases!
 +
 -----------------------------------------------------------------------------
 mkdir distri
 cd distri
-git clone git@github.com:os-autoinst/os-autoinst-distri-opensuse.git opensuse
-cd opensuse
+git clone git@github.com:os-autoinst/os-autoinst-distri-opensuse.git
```

Comment:
I would really recommend to keep the alias "opensuse" intact as this is commonly used when referring to the test distribution with the `DISTRI` variable